src/openradardata_validator/odim2ordmsg.py

```diff
@@ -4,13 +4,17 @@
 import os
 from datetime import datetime
 from pathlib import Path
-from typing import Any
+from typing import Any, Union
```

Comment:
Then this import will not be necessary
```suggestion
from typing import Any
```